Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -415,6 +415,15 @@ class BrowserViewController: UIViewController {
             // Reset previous crash state
             activeCrashReporter?.resetPreviousCrashState()
 
+            // Only ask to restore tabs from a crash if we had non-home tabs or tabs with some kind of history in them
+            guard let tabsToRestore = tabManager.tabsToRestore() else { return }
+            let onlyNoHistoryTabs = !tabsToRestore.every { $0.sessionData?.urls.count > 1 }
+            if onlyNoHistoryTabs {
+                let tab = tabManager.addTab()
+                tabManager.selectTab(tab)
```

Comment:
Are you implying we should autoselect the created tab if we add a tab and it's the only one available? Not sure if that's better than selected the tab after it's been created. Before when we had only normal tabs it made sense since we could never have zero tabs but for private mode we can have a situation where we have none to display in the tab tray.

I can add a convenience method though named 'addTabAndSelect'.